tests/test_selector_csstranslator.py

```diff
@@ -44,9 +44,7 @@
 '''
 
 
-class TranslatorMixinTest(unittest.TestCase):
-
-    tr_cls = HTMLTranslator
+class TranslatorTestMixin():
```

Comment:
```suggestion
class TranslatorTestMixin:
```

If the above works, it is better :)